### PR TITLE
[4.0] Smart Search help strings

### PR DESCRIPTION
**Pull request by brianteeman**
Looks like I removed a few _DESC strings that were being used as the tip for the search filter in the com_finder tables

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_finder.ini [hunk 1]
```diff
@@ -174,8 +174,12 @@ COM_FINDER_QUERY_OPERATOR_NOT="Not"
 COM_FINDER_QUERY_OPERATOR_OR="Or"
 COM_FINDER_SEARCH_FILTER_SEARCH_LABEL="Search Filters"
 COM_FINDER_SEARCH_IN_PHRASE_LABEL="Filter by searched phrase"
-COM_FINDER_SEARCH_IN_PHRASE_DESC="Search for the logged phrase of a search"
+COM_FINDER_SEARCH_IN_PHRASE_DESC="Search for the logged phrase of a search."
 COM_FINDER_SEARCH_LABEL="Search %s:"
+COM_FINDER_SEARCH_FILTER_SEARCH_DESC="Search in filter title."
+COM_FINDER_SEARCH_FILTER_SEARCH_LABEL="Search Filters"
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.com_finder.ini:176]: Duplicate line 175.

File: administrator/language/en-GB/en-GB.com_finder.ini [hunk 2]
```diff
@@ -174,8 +174,12 @@ COM_FINDER_QUERY_OPERATOR_NOT="Not"
...
+COM_FINDER_SEARCH_LABEL="Search %s:"
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.com_finder.ini:179]: Duplicate line 178.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 0c254352d91689b01ce4ce34698b03752d275e0e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26279">issues.joomla.org/tracker/joomla-cms/26279</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thanks for that

**Pull request merged**